Src/MoneyFox.Core/Queries/Payments/GetPaymentsForAccountId/GetPaymentsForAccountIdQuery.cs

```diff
@@ -66,6 +70,11 @@ public Handler(IContextAdapter contextAdapter)
                     paymentQuery = paymentQuery.AreRecurring();
                 }
 
+                if (request.PaymentTypeFilter != -1)
+                {
+                    paymentQuery = paymentQuery.IsPaymentType((PaymentType)request.PaymentTypeFilter);
```

Comment:
This might be worth a new unit test in the GetPaymentsForAccountIdQueryTests :)

Reply:
Definitely! I'll add a unit test.